src/common/service/proxy.js

```diff
@@ -10,8 +10,8 @@ export default class Proxy {
 
     this._proxy = createHoxy();
     this._proxy.intercept('response-sent', this._onResponseSent.bind(this));
-    this._proxy.intercept('request', this._onInterceptRequest.bind(this));
-    this._proxy.intercept('response', this._onInterceptResponse.bind(this));
+    this._proxy.intercept({phase: 'request', as: 'string'}, this._onInterceptRequest.bind(this));
+    this._proxy.intercept({phase: 'response', as: 'string'} , this._onInterceptResponse.bind(this));
```

Comment:
Small nitpick, and the reason why the build failed: there's a space between the `}` and comma here :slightly_smiling_face: